### PR TITLE
FRW-8883 Fixed customer login so that email confirmation is checked and errors are correctly reported.

### PR DESCRIPTION
**Pull request by aleksandr-velikanov**
Ticket:  https://spryker.atlassian.net/browse/FRW-8883

##### Change log

Improvements

- Added a test for customer login with an unconfirmed email.
- Added a keyword to log in a customer into Yves and expect an error.

### CODE REVIEW

File: tests/ui/suite/customers/customer.robot [hunk 1]
```diff
@@ -396,3 +396,15 @@ Shopping_List_Contains_Offers
     [Teardown]    Run Keywords    Yves: login on Yves with provided credentials:    ${yves_company_user_buyer_email}
     ...    AND    Yves: check if cart is not empty and clear it
     ...    AND    Yves: delete 'Shopping List' with name:    shoppingListName${random}
+
+Email_Confirmation
+    [Documentation]    Check that a new user can cannot login if the email is not verified
+    Register a new customer with data:
+    ...    || salutation | first name | last name | e-mail                       | password                                        ||
+    ...    || Mr.        | New        | User      | sonia+${random}@spryker.com  | Ps${random_str_password}!5${random_id_password} ||
```
Comment (AntonShopin) [inline tests/ui/suite/customers/customer.robot:404]: Please change the email address that you use in the test. The problem is that there is another test that creates a new customer with `sonia+${random}@spryker.com`, so in this test most probably registration will fail as the email address is already in use. 

Change it on smth unique, like `sonia+fail${random}@spryker.com`

**Review by AntonShopin — COMMENTED** [review]
Please create the same test for demoshops (b2b, b2c, b2b-mp, b2c-mp) as well

**Review by kraal-spryker — APPROVED** [review]

**Review by AntonShopin — APPROVED** [review]

File: tests/ui/mp_b2c/customers/customer.robot [hunk 2]
```diff
@@ -219,3 +219,14 @@ Update_Customer_Data
     ...    AND    Zed: update customer profile data:
     ...    || email                     | salutation | first name                     | last name                     ||
     ...    || ${yves_second_user_email} | Mr         | ${yves_second_user_first_name} | ${yves_second_user_last_name} ||
+
+Email_Confirmation
+    [Documentation]    Check that a new user can cannot login if the email is not verified
```
Comment (asmarovydlo) [inline tests/ui/mp_b2c/customers/customer.robot:224]: So, can or can't?

File: tests/ui/b2c/customers/customer.robot [hunk 3]
```diff
@@ -200,3 +200,14 @@ Update_Customer_Data
     ...    AND    Zed: update customer profile data:
     ...    || email                     | salutation | first name                     | last name                     ||
     ...    || ${yves_second_user_email} | Mr         | ${yves_second_user_first_name} | ${yves_second_user_last_name} ||
+
+Email_Confirmation
+    [Documentation]    Check that a new user can cannot login if the email is not verified
```
Comment (asmarovydlo) [inline tests/ui/b2c/customers/customer.robot:205]: So, can or can't?

File: tests/ui/suite/customers/customer.robot [hunk 4]
```diff
@@ -396,3 +396,15 @@ Shopping_List_Contains_Offers
     [Teardown]    Run Keywords    Yves: login on Yves with provided credentials:    ${yves_company_user_buyer_email}
     ...    AND    Yves: check if cart is not empty and clear it
     ...    AND    Yves: delete 'Shopping List' with name:    shoppingListName${random}
+
+Email_Confirmation
+    [Documentation]    Check that a new user can cannot login if the email is not verified
```
Comment (asmarovydlo) [inline tests/ui/suite/customers/customer.robot:401]: So, can or can't?

**Review by asmarovydlo — CHANGES_REQUESTED** [review]

**Review by asmarovydlo — APPROVED** [review]

**Pull request merged**